xarray/tests/test_dataset.py

```diff
@@ -178,24 +178,22 @@ def test_repr(self):
         data = create_test_data(seed=123)
         data.attrs["foo"] = "bar"
         # need to insert str dtype at runtime to handle both Python 2 & 3
```

Comment:
```suggestion
```

Reply:
we still need  this for supporting different endiannesses